spec/unit/provider/package/dpkg_spec.rb

```diff
@@ -52,8 +51,8 @@
   end
 
   before(:each) do
-    allow(provider).to receive(:shell_out!).with("dpkg-deb -W #{new_resource.source}", timeout: 900).and_return(dpkg_deb_status)
-    allow(provider).to receive(:shell_out).with("dpkg -s #{package}", timeout: 900).and_return(dpkg_s_status)
+    allow(provider).to receive(:shell_out!).with("dpkg-deb -W #{source}", timeout: 900).and_return(dpkg_deb_status)
```

Comment:
I know you can do the opposite - you can have a broad `allow(thing).to receive(:foo)` and then narrow in with specific expects, EG `expect(thing).to_not receive(:foo).with('bar')` or `expect(thing).to receive(:foo).exactly(1000).times`